generators/entity-server/templates/src/main/java/package/domain/Entity.java.jhi.javax_persistence.ejs

```diff
@@ -0,0 +1,130 @@
+<%#
+ Copyright 2013-2021 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+<%#
+ This is a partial file, it will be merged into to root template if available.
+ EJS partials will process % delimiter tags in template and & delimiter tags in the merge process.
+-%>
+<&_ if (partialImportSection) { -&>
+import javax.persistence.*;
+<%_ if ((databaseTypePostgres && fields.find(field => !field.transient && field.blobContentTypeText)) || (fieldsContainUUID && (databaseTypeMysql || databaseTypeMariadb))) { _%>
```

Comment:
I really don't get it

a good practice is to extract a variable from complex conditions so as to give them meaning

can you do it in every modified please? this would make the review way easier :) thanks

Reply:
Let’s keep this way for this PR.
It’s focused on the fragments support.
Not rewrite every file for best practices.

Reply:
Sorry but no. This MR is too complicated to be reviewed, even for me.